### ISSUE TITLE
[Feature] Identify multiple default branches

### ISSUE THREAD

**Issue by NiallJude** [tj-actions/branch-name#112]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

Hi there,

I am wondering if this works with multiple branches identified as default. What I am trying to do is run Step X on Main and Develop runs only. However, only one branch at a time can be chosen as "Default" in settings. So I am probably missing some other setting than just the default check.

Basically I want two stages:
- Do X on Main and Develop.
- Not on Main and Develop? Do Y.

### Describe the solution you'd like?

Basically, I would just like to know if it is possible already.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by NiallJude** [comment]
Got a solution implemented that works as below:

```if: steps.branch-name.outputs.current_branch == 'main' || steps.branch-name.outputs.current_branch == 'develop'```

However, I am leaving this open in case there is a more elegant way you know which I am missing.
Thanks for what you do! Star added.

**Comment by jackton1** [comment]
@NiallJude You have the correct implementation based on the fact that only the default branch can be detected.

An alternative approach would be to use:

```yaml
if: steps.branch-name.outputs.is_default == 'true' || steps.branch-name.outputs.current_branch == 'develop'
```
